fdbclient/include/fdbclient/DatabaseContext.h

```diff
@@ -207,6 +207,16 @@ struct KeyRangeLocationInfo {
 	  : tenantEntry(tenantEntry), range(range), locations(locations) {}
 };
 
+struct OverlappingChangeFeedsInfo {
+	Arena arena;
+	VectorRef<OverlappingChangeFeedEntry> feeds;
+	// would prefer to use key range map but it complicates copy/move constructors
+	std::vector<std::pair<KeyRangeRef, Version>> metadataVersions;
```

Comment:
is this "metadata version" the same as the one in `GetReadVersionReply`? If not, I'd suggest rename it to avoid potential confusion.

Reply:
It is different, thanks for the comments.